build

```diff
@@ -8,4 +8,10 @@
 CHART="$(ls charts)"
 CHART_META="charts/${CHART}/Chart.yaml"
 
-make images-kind-load VERSION="$(yq .version "${CHART_META}")"
+if [[ "$(uname)" == "Darwin" ]]; then
```

Comment:
I mean, given

```
[Wed 19 Jun 11:34:08 BST 2024] simon@symphony ~/src/github.com/unikorn-cloud/scripts which gmake
/usr/bin/gmake
```

I wouldn't be opposed to just changing the command rather than having a sprawling mess of control flow

Reply:
Well now that you mention it, what I actually want to do is to state that developers on macOS should use Colima, and then this build command becomes `gmake build images`.

Reply:
Well then, given `kubeclt version` doesn't give away the platform, and a special flag will be intrusive to someone, I'd consider switching the command based on a config file, say `~/.config/unikorn/scipts.yaml`.  That would look like...
```yaml
platform: kind
```
Then do
```bash
MAKE_TARGET=images

if [[ "$(yq .platform /.config/unikorn/scipts.yaml)" == "kind" ]]; then
  MAKE_TARGET=images-kind-load
fi

gmake "${MAKE_TARGET}" ...blah
```